specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/managedHsm.json

```diff
@@ -0,0 +1,624 @@
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedHsmSku": {
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "SKU Family of the managed HSM Pool",
+          "enum": [
+            "B"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedHsmSkuFamily",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "SKU of the managed HSM Pool",
+          "enum": [
+            "Standard_B1",
+            "Custom_B32"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedHsmSkuName",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "SKU details",
+      "required": [
+        "name",
+        "family"
+      ]
+    },
+    "ManagedHsmProperties": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The Azure Active Directory tenant ID that should be used for authenticating requests to the managed HSM pool."
+        },
+        "securityDomainId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The security domain id that should be used for restoring data from this managed HSM pool to a new managed hsm pool."
+        },
+        "initialAdminObjectIds": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Array of initial administrators object ids for this managed hsm pool."
+        },
+        "hsmPoolUri": {
+          "type": "string",
+          "description": "The URI of the managed hsm pool for performing operations on keys and secrets."
+        },
+        "enableSoftDelete": {
+          "type": "boolean",
+          "default": true,
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this managed HSM pool. If it's not set to any value(true or false) when creating new managed HSM pool, it will be set to true by default. Once set to true, it cannot be reverted to false."
+        },
+        "softDeleteRetentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "default": 90,
+          "description": "softDelete data retention days. It accepts >=7 and <=90."
+        },
+        "enablePurgeProtection": {
+          "type": "boolean",
+          "description": "Property specifying whether protection against purge is enabled for this managed HSM pool. Setting this property to true activates protection against purge for this managed HSM pool and its content - only the Managed HSM service may initiate a hard, irrecoverable deletion. The setting is effective only if soft delete is also enabled. Enabling this functionality is irreversible - that is, the property does not accept false as its value."
```

Comment:
In this description you say: "Enabling this functionality is irreversible - that is, the property does not accept false as its value."

Irreversible does not mean we cannot set false as the value right? I mean even if it is irreversible we can still set this property during intial creation right? If so, i recommend removing this part: " - that is, the property does not accept false as its value."

Reply:
Removed.